CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   - `sequence`
 - Added support for registering and calling Stored Procedures with `TABLE` return type.
 - Added support for parameter `length` in `StringType()` to specify the maximum number of characters that can be stored by the column.
+- Added support to run on python 3.9 runtime environment.
```

Comment:
How about this?

Added support for the Python 3.9 runtime environment.

Reply:
We might not have to add this. Let me confirm with PMs and let you know.

Reply:
I think support for Python 3.9 as a Public Preview feature is being announced as part of the 7.19 BCR release on May 30/31 (I know this because I wrote the BCR release note for it). 

As written, this sentence doesn't make it clear that support is not GA yet. It might be good to leave it out. Whatever the PM says is great.